src/commands/referral/referralRandom.ts

```diff
@@ -35,17 +34,17 @@ export const execute: CommandHandler = async (interaction) => {
       service: {
         contains: service,
       },
+      expiry_date: {
```

Comment:
This just leaves the filtering to the DB/ORM, we don't need to do a filter step ourselves.